### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by kambysese**
@agramfort please review.

### CODE REVIEW

**Comment by dengemann** [comment]
Looks good to me!

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,6 @@ doc/samples
 cover
 
 *.orig
+
+# PyCharm
+.idea/**
```
Comment (jaeilepp) [inline .gitignore]: Add a newline in the end.

File: mne/decoding/base.py [hunk 2]
```diff
@@ -352,7 +352,8 @@ def plot_patterns(self, info, times=None, ch_type=None, layout=None,
         ch_type : 'mag' | 'grad' | 'planar1' | 'planar2' | 'eeg' | None
             The channel type to plot. For 'grad', the gradiometers are
             collected in pairs and the RMS for each pair is plotted.
-            If None, then channels are chosen in the order given above.
+            If None, then first available channel type from order given
++           above is used. Defaults to None.
```
Comment (jaeilepp) [inline mne/decoding/base.py]: Remove extra +

**Comment by agramfort** [comment]
merged by rebase after cleaning history.

thx @kambysese !

**Comment by kambysese** [comment]
thx @agramfort. sorry about the history, I'm a bit of a masochist when it comes to commit histories. just ask @Eric89GXL :)

**Pull request closed without merging**